conduit/src/Data/Conduit/Internal/Conduit.hs

```diff
@@ -763,12 +769,15 @@ src $$ sink = do
 {-# RULES "conduit: =$ is =$=" (=$) = (=$=) #-}
 {-# DEPRECATED (=$) "Use .|" #-}
 
--- | Fusion operator, combining two @Conduit@s together into a new @Conduit@.
---
--- Both @Conduit@s will be closed when the newly-created @Conduit@ is closed.
+-- | Combine two @Conduit@s together into a new @Conduit@ (aka 'fuse').
```

Comment:
Given that this is already deprecated, how about just documenting as "deprecated operator for `.|`"

Reply:
Since this is what the depreciation remark already says, I've changed the wording to "Deprecated fusion operator". Together with the depreciation remark that should give a clear picture.